paasta_tools/cli/cmds/validate.py

```diff
@@ -258,6 +262,13 @@ def add_subparser(subparsers):
         required=False,
         help="Service that you want to validate. Like 'example_service'.",
     ).completer = lazy_choices_completer(list_services)
+    validate_parser.add_argument(
+        "-v",
+        "--verbose",
+        action="store_true",
+        required=False,
+        help="Display verbose output. This shows the next few cron runs scheduled.",
```

Comment:
thoughts on something like
```suggestion
        help="Toggle to display additional validation messages for humans.",
```

if we ever add any other verbose output, we'd have to further update this help text if we leave this as-is